ledcontroller/__init__.py

```diff
@@ -440,3 +453,26 @@ def batch_run(self, *commands):
                 args = command[1:]
                 cmd(*args)
         self.repeat_commands = original_retries
+
+
+
+def rgb_to_hue(red, green, blue):
```

Comment:
I would prefer having this as a `@classmethod` under `LedController`, instead of as a separate method.